internal/olm/operator/operator.go

```diff
@@ -97,6 +100,8 @@ func (c *OLMCmd) AddToFlagSet(fs *pflag.FlagSet) {
 			"These supplement or override defaults generated by run/cleanup")
 	fs.DurationVar(&c.Timeout, "timeout", defaultTimeout,
 		prefix+"Time to wait for the command to complete before failing")
+	fs.StringVar(&c.OLMNamespace, "olm-namespace", olm.DefaultNamespace,
+		prefix+"OLM Namespace")
 }
 
 func (c *OLMCmd) validate() error {
```

Comment:
Validate whether `OLMCmd.OLMNamespace` is set.

Reply:
`OLMNamespace` is currently optional, and it defaults to `olm.DefaultNamespace`(olm) , not sure what validation should be done?, can you pls provide an example if i'm missing anything

Reply:
It defaults to `olm.DefaultNamespace` in its flags, but what if `OLMCmd` is populated directly in code? `OLMCmd.OLMNamespace` isn't defaulted anywhere AFAIK, and is used directly by `operatorManager` methods.

Reply:
ah yes. added it now.

Reply:
Is not missing address this one?